docs/storage.md

```diff
@@ -61,17 +61,20 @@ A Prometheus server's data directory looks something like this:
 Note that a limitation of local storage is that it is not clustered or
 replicated. Thus, it is not arbitrarily scalable or durable in the face of
 drive or node outages and should be managed like any other single node
-database. 
+database.
 
 [Snapshots](querying/api.md#snapshot) are recommended for backups. Backups 
 made without snapshots run the risk of losing data that was recorded since 
-the last WAL sync, which typically happens every two hours. With proper
-architecture, it is possible to retain years of data in local storage.
-
-Alternatively, external storage may be used via the
-[remote read/write APIs](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).
-Careful evaluation is required for these systems as they vary greatly in durability,
-performance, and efficiency.
+the last WAL sync, which typically happens every two hours, covering
+the last three hours of samples. Excluding the WAL files (the `wal/`
+directory in `storage.tsdb.path`) on backup or restore will ensure a
+coherent backup, in any case, at the cost of losing that time range.
+
+Alternatively, external storage may be used via the [remote read/write
+APIs](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).
+Careful evaluation is required for these systems as they vary greatly
+in durability, performance, and efficiency. With proper architecture,
+it is possible to retain years of data in local storage.
```

Comment:
I think this last bit was originally saying you don't need external storage, so moving it here changes the meaning.

Reply:
i moved it back, is it okay now?